### ISSUE TITLE
Problem returning excerpt in meta Desc

### ISSUE THREAD

**Issue by Mindgames** [ashhitch/wp-graphql-yoast-seo#28]
Using %%excerpt%% as the default description on a custom post type don't return the generated value. In editor & WP frontend it works
![image](https://user-images.githubusercontent.com/2171273/80651860-f9cda780-8a76-11ea-8616-e9bbc1709013.png)

in my GQL query i get 
```
"seo": {
        "title": "Stable Baselines",
        "metaDesc": "%%excerpt%%",
```
Any ideas?

**Comment by Mindgames** [comment]
Tested to set the meta description manually for the post + removed any preset values in the Yoast SEO content settings. However it return same value in `metaDesc` despite this being set individually. 
```
{
  "data": {
    "ecosystems": {
      "edges": [
        {
          "node": {
            "id": "ZWNvc3lzdGVtOjEwNw==",
            "title": "Stable Baselines",
            "date": "2020-04-29T14:52:29",
            "slug": "stable-baselines",
            "content": "\n<p><a href=\"https:\/\/github.com\/hill-a\/stable-baselines\">Stable Baselines<\/a>&nbsp;is a set of improved implementations of Reinforcement Learning (RL) algorithms based on OpenAI&nbsp;<a href=\"https:\/\/github.com\/openai\/baselines\">Baselines<\/a>.<\/p>\n\n\n\n<figure class=\"wp-block-image size-large\"><img src=\"http:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/stable-Baselines.png\" alt=\"Stable Baselines\" class=\"wp-image-108\" srcset=\"https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/stable-Baselines.png 616w, https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/stable-Baselines-300x262.png 300w\" sizes=\"(max-width: 616px) 100vw, 616px\" \/><\/figure>\n\n\n\n<h2>Main differences with OpenAI Baselines<\/h2>\n\n\n\n<p>This toolset is a fork of OpenAI Baselines, with a major structural refactoring, and code cleanups:<\/p>\n\n\n\n<ul><li>Unified structure for all algorithms<\/li><li>PEP8 compliant (unified code style)<\/li><li>Documented functions and classes<\/li><li>More tests &amp; more code coverage<\/li><li>Additional algorithms: SAC and TD3 (+ HER support for DQN, DDPG, SAC and TD3)<\/li><\/ul>\n\n\n\n<figure class=\"wp-block-image size-large is-resized\"><img src=\"https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/Screenshot-2020-04-29-at-19.05.01-1024x707.png\" alt=\"\" class=\"wp-image-119\" width=\"618\" height=\"426\" srcset=\"https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/Screenshot-2020-04-29-at-19.05.01-1024x707.png 1024w, https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/Screenshot-2020-04-29-at-19.05.01-300x207.png 300w, https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/Screenshot-2020-04-29-at-19.05.01-768x530.png 768w, https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/Screenshot-2020-04-29-at-19.05.01.png 1112w\" sizes=\"(max-width: 618px) 100vw, 618px\" \/><\/figure>\n\n\n\n<h2>Try Stable Baselines with this examples<\/h2>\n\n\n\n<p>We created a couple of example notebooks and also including notebooks made by the folks behind stable baselines<\/p>\n\n\n\n<h3>BipedalWalker-V3 with Tensorboard &amp; Video<\/h3>\n\n\n\n<p>A Complete setup in Google Colab that include tensorboard and on top of that generate out videos to allow you to follow the progress. <\/p>\n\n\n\n<h4>BipedalWalker-V3 with PPO2 Algorithm<\/h4>\n\n\n\n<p>The Proximal Policy Optimization algorithm combines ideas from A2C (having multiple workers) and TRPO (it uses a trust region to improve the actor).  Try out our Notebook using PPO2<\/p>\n\n\n\n<figure class=\"wp-block-image colab\"><a href=\"https:\/\/colab.research.google.com\/drive\/1kl7124cI89-yn-Stio8GiLpJV0jec6Kt\" target=\"_blank\" rel=\"noreferrer noopener\"><img src=\"https:\/\/camo.githubusercontent.com\/52feade06f2fecbf006889a904d221e6a730c194\/68747470733a2f2f636f6c61622e72657365617263682e676f6f676c652e636f6d2f6173736574732f636f6c61622d62616467652e737667\" alt=\"BipedalWalker-V3 with PPO2 Algorithm\"\/><\/a><\/figure>\n\n\n\n<h4>BipedalWalker-V3 with TD3 Algorithm<\/h4>\n\n\n\n<p>Notebook using Twin Delayed DDPG (TD3)<\/p>\n\n\n\n<figure class=\"wp-block-image is-resized colab\"><a href=\"https:\/\/colab.research.google.com\/drive\/1Zyn9Q_Gf3KnVIhdl9t2ond5IjJNaTriL\" target=\"_blank\" rel=\"noreferrer noopener\"><img src=\"https:\/\/camo.githubusercontent.com\/52feade06f2fecbf006889a904d221e6a730c194\/68747470733a2f2f636f6c61622e72657365617263682e676f6f676c652e636f6d2f6173736574732f636f6c61622d62616467652e737667\" alt=\"BipedalWalker-V3 with TD3 Algorithm\" width=\"117\" height=\"20\"\/><\/a><\/figure>\n\n\n\n<h3 class=\"has-light-gray-color has-text-color\">Notebooks by Stable baselines<\/h3>\n\n\n\n<ul><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/stable_baselines_getting_started.ipynb\">Getting Started<\/a><\/li><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/saving_loading_dqn.ipynb\">Training, Saving, Loading<\/a><\/li><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/multiprocessing_rl.ipynb\">Multiprocessing<\/a><\/li><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/monitor_training.ipynb\">Monitor Training and Plotting<\/a><\/li><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/atari_games.ipynb\">Atari Games<\/a><\/li><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/breakout.ipynb\">Breakout<\/a>&nbsp;(trained agent included)<\/li><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/stable_baselines_her.ipynb\">Hindsight Experience Replay<\/a><\/li><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/rl-baselines-zoo.ipynb\">RL Baselines zoo<\/a><\/li><\/ul>\n\n\n\n<p><a href=\"\"><\/a><\/p>\n\n\n\n<p>Github repository:&nbsp;<a href=\"https:\/\/github.com\/hill-a\/stable-baselines\">https:\/\/github.com\/hill-a\/stable-baselines<\/a><\/p>\n\n\n\n<p><\/p>\n\n\n\n<p><\/p>\n",
            "assets": {
              "description": "Stable Baselines is a set of improved implementations of Reinforcement Learning (RL) algorithms based on OpenAI Baselines.",
              "thumbnail": {
                "sourceUrl": "https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/stable-Baselines.png"
              },
              "level": "Normal"
            },
            "seo": {
              "title": "Stable Baselines",
              "metaDesc": "This toolset is a fork of OpenAI Baselines, with a major structural refactoring, and code cleanups"
            }
          }
        },
        {
          "node": {
            "id": "ZWNvc3lzdGVtOjExMA==",
            "title": "OpenAI Gym",
            "date": "2020-04-28T15:54:34",
            "slug": "openai-gym",
            "content": "\n<figure class=\"wp-block-image alignfull size-large\"><img src=\"http:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/openaigymhero-1024x320.jpeg\" alt=\"\" class=\"wp-image-111\" srcset=\"https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/openaigymhero-1024x320.jpeg 1024w, https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/openaigymhero-300x94.jpeg 300w, https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/openaigymhero-768x240.jpeg 768w, https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/openaigymhero.jpeg 1500w\" sizes=\"(max-width: 1024px) 100vw, 1024px\" \/><\/figure>\n\n\n\n<div style=\"height:29px\" aria-hidden=\"true\" class=\"wp-block-spacer\"><\/div>\n\n\n\n<p>Gym is a toolkit for developing and comparing reinforcement learning algorithms. It supports teaching agents everything from&nbsp;walking&nbsp;to playing games like&nbsp;Pong&nbsp;or&nbsp;Pinball.<\/p>\n\n\n\n<p><a href=\"https:\/\/gym.openai.com\/docs\/\">https:\/\/gym.openai.com\/docs\/<\/a><\/p>\n\n\n\n<p><a href=\"https:\/\/github.com\/openai\/gym\">https:\/\/github.com\/openai\/gym<\/a><\/p>\n",
            "assets": {
              "description": "Gym is a toolkit for developing and comparing reinforcement learning algorithms. ",
              "thumbnail": {
                "sourceUrl": "https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/openai.png"
              },
              "level": "Hard"
            },
            "seo": {
              "title": "Stable Baselines",
              "metaDesc": "This toolset is a fork of OpenAI Baselines, with a major structural refactoring, and code cleanups"
            }
          }
        }
      ]
    }
  }
}
```

**Comment by Mindgames** [comment]
**However** 
When running directly towards the custom post it works 

```
{
  "data": {
    "ecosystemBy": {
      "id": "ZWNvc3lzdGVtOjExMA==",
      "title": "OpenAI Gym",
      "date": "2020-04-28T15:54:34",
      "slug": "openai-gym",
      "content": "\n<figure class=\"wp-block-image alignfull size-large\"><img src=\"http:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/openaigymhero-1024x320.jpeg\" alt=\"\" class=\"wp-image-111\" srcset=\"https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/openaigymhero-1024x320.jpeg 1024w, https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/openaigymhero-300x94.jpeg 300w, https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/openaigymhero-768x240.jpeg 768w, https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/openaigymhero.jpeg 1500w\" sizes=\"(max-width: 1024px) 100vw, 1024px\" \/><\/figure>\n\n\n\n<div style=\"height:29px\" aria-hidden=\"true\" class=\"wp-block-spacer\"><\/div>\n\n\n\n<p>Gym is a toolkit for developing and comparing reinforcement learning algorithms. It supports teaching agents everything from&nbsp;walking&nbsp;to playing games like&nbsp;Pong&nbsp;or&nbsp;Pinball.<\/p>\n\n\n\n<p><a href=\"https:\/\/gym.openai.com\/docs\/\">https:\/\/gym.openai.com\/docs\/<\/a><\/p>\n\n\n\n<p><a href=\"https:\/\/github.com\/openai\/gym\">https:\/\/github.com\/openai\/gym<\/a><\/p>\n",
      "assets": {
        "description": "Gym is a toolkit for developing and comparing reinforcement learning algorithms. ",
        "thumbnail": {
          "sourceUrl": "https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/openai.png"
        },
        "level": "Hard"
      },
      "seo": {
        "title": "OpenAI Gym",
        "metaDesc": "Gym is a toolkit for developing and comparing reinforcement learning algorithms.",
        "opengraphTitle": "",
        "opengraphDescription": "",
        "opengraphImage": null,
        "twitterTitle": "",
        "twitterDescription": "",
        "twitterImage": null
      }
    }
  }
}
```
And the other one 
```
{
  "data": {
    "ecosystemBy": {
      "id": "ZWNvc3lzdGVtOjEwNw==",
      "title": "Stable Baselines",
      "date": "2020-04-29T14:52:29",
      "slug": "stable-baselines",
      "content": "\n<p><a href=\"https:\/\/github.com\/hill-a\/stable-baselines\">Stable Baselines<\/a>&nbsp;is a set of improved implementations of Reinforcement Learning (RL) algorithms based on OpenAI&nbsp;<a href=\"https:\/\/github.com\/openai\/baselines\">Baselines<\/a>.<\/p>\n\n\n\n<figure class=\"wp-block-image size-large\"><img src=\"http:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/stable-Baselines.png\" alt=\"Stable Baselines\" class=\"wp-image-108\" srcset=\"https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/stable-Baselines.png 616w, https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/stable-Baselines-300x262.png 300w\" sizes=\"(max-width: 616px) 100vw, 616px\" \/><\/figure>\n\n\n\n<h2>Main differences with OpenAI Baselines<\/h2>\n\n\n\n<p>This toolset is a fork of OpenAI Baselines, with a major structural refactoring, and code cleanups:<\/p>\n\n\n\n<ul><li>Unified structure for all algorithms<\/li><li>PEP8 compliant (unified code style)<\/li><li>Documented functions and classes<\/li><li>More tests &amp; more code coverage<\/li><li>Additional algorithms: SAC and TD3 (+ HER support for DQN, DDPG, SAC and TD3)<\/li><\/ul>\n\n\n\n<figure class=\"wp-block-image size-large is-resized\"><img src=\"https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/Screenshot-2020-04-29-at-19.05.01-1024x707.png\" alt=\"\" class=\"wp-image-119\" width=\"618\" height=\"426\" srcset=\"https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/Screenshot-2020-04-29-at-19.05.01-1024x707.png 1024w, https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/Screenshot-2020-04-29-at-19.05.01-300x207.png 300w, https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/Screenshot-2020-04-29-at-19.05.01-768x530.png 768w, https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/Screenshot-2020-04-29-at-19.05.01.png 1112w\" sizes=\"(max-width: 618px) 100vw, 618px\" \/><\/figure>\n\n\n\n<h2>Try Stable Baselines with this examples<\/h2>\n\n\n\n<p>We created a couple of example notebooks and also including notebooks made by the folks behind stable baselines<\/p>\n\n\n\n<h3>BipedalWalker-V3 with Tensorboard &amp; Video<\/h3>\n\n\n\n<p>A Complete setup in Google Colab that include tensorboard and on top of that generate out videos to allow you to follow the progress. <\/p>\n\n\n\n<h4>BipedalWalker-V3 with PPO2 Algorithm<\/h4>\n\n\n\n<p>The Proximal Policy Optimization algorithm combines ideas from A2C (having multiple workers) and TRPO (it uses a trust region to improve the actor).  Try out our Notebook using PPO2<\/p>\n\n\n\n<figure class=\"wp-block-image colab\"><a href=\"https:\/\/colab.research.google.com\/drive\/1kl7124cI89-yn-Stio8GiLpJV0jec6Kt\" target=\"_blank\" rel=\"noreferrer noopener\"><img src=\"https:\/\/camo.githubusercontent.com\/52feade06f2fecbf006889a904d221e6a730c194\/68747470733a2f2f636f6c61622e72657365617263682e676f6f676c652e636f6d2f6173736574732f636f6c61622d62616467652e737667\" alt=\"BipedalWalker-V3 with PPO2 Algorithm\"\/><\/a><\/figure>\n\n\n\n<h4>BipedalWalker-V3 with TD3 Algorithm<\/h4>\n\n\n\n<p>Notebook using Twin Delayed DDPG (TD3)<\/p>\n\n\n\n<figure class=\"wp-block-image is-resized colab\"><a href=\"https:\/\/colab.research.google.com\/drive\/1Zyn9Q_Gf3KnVIhdl9t2ond5IjJNaTriL\" target=\"_blank\" rel=\"noreferrer noopener\"><img src=\"https:\/\/camo.githubusercontent.com\/52feade06f2fecbf006889a904d221e6a730c194\/68747470733a2f2f636f6c61622e72657365617263682e676f6f676c652e636f6d2f6173736574732f636f6c61622d62616467652e737667\" alt=\"BipedalWalker-V3 with TD3 Algorithm\" width=\"117\" height=\"20\"\/><\/a><\/figure>\n\n\n\n<h3 class=\"has-light-gray-color has-text-color\">Notebooks by Stable baselines<\/h3>\n\n\n\n<ul><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/stable_baselines_getting_started.ipynb\">Getting Started<\/a><\/li><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/saving_loading_dqn.ipynb\">Training, Saving, Loading<\/a><\/li><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/multiprocessing_rl.ipynb\">Multiprocessing<\/a><\/li><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/monitor_training.ipynb\">Monitor Training and Plotting<\/a><\/li><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/atari_games.ipynb\">Atari Games<\/a><\/li><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/breakout.ipynb\">Breakout<\/a>&nbsp;(trained agent included)<\/li><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/stable_baselines_her.ipynb\">Hindsight Experience Replay<\/a><\/li><li><a href=\"https:\/\/colab.research.google.com\/github\/Stable-Baselines-Team\/rl-colab-notebooks\/blob\/master\/rl-baselines-zoo.ipynb\">RL Baselines zoo<\/a><\/li><\/ul>\n\n\n\n<p><a href=\"\"><\/a><\/p>\n\n\n\n<p>Github repository:&nbsp;<a href=\"https:\/\/github.com\/hill-a\/stable-baselines\">https:\/\/github.com\/hill-a\/stable-baselines<\/a><\/p>\n\n\n\n<p><\/p>\n\n\n\n<p><\/p>\n",
      "assets": {
        "description": "Stable Baselines is a set of improved implementations of Reinforcement Learning (RL) algorithms based on OpenAI Baselines.",
        "thumbnail": {
          "sourceUrl": "https:\/\/data.dllglobal.com\/wp-content\/uploads\/2020\/04\/stable-Baselines.png"
        },
        "level": "Normal"
      },
      "seo": {
        "title": "Stable Baselines",
        "metaDesc": "This toolset is a fork of OpenAI Baselines, with a major structural refactoring, and code cleanups",
        "opengraphTitle": "",
        "opengraphDescription": "",
        "opengraphImage": null,
        "twitterTitle": "",
        "twitterDescription": "",
        "twitterImage": null
      }
    }
  }
}
```

**Comment by ashhitch** [comment]
Hey @Mindgames can I ask which versions of: WP, WpGraphql and Yoast you are using?
These are the results im getting:

<img width="1042" alt="Screenshot 2020-04-30 at 08 17 41" src="https://user-images.githubusercontent.com/2438290/80682890-39b67e00-8abb-11ea-9e52-fe1fa2c09718.png">

<img width="647" alt="image" src="https://user-images.githubusercontent.com/2438290/80682937-4dfa7b00-8abb-11ea-9fc3-b0d65130bd82.png">

**Comment by ashhitch** [comment]
@Mindgames  I have just installed the RC of Yoast 14 and can see it has breaking changes for my plugin., I will start to look into this., if you were to install version 13 of the Yoast plugin it should work for you.

**Comment by Mindgames** [comment]
@ashhitch Downgrading to YOAST 13 did solve the issue.  Thanks for the quick response

**Comment by ashhitch** [comment]
Good to hear, I m working on a new release that should fix this from v14.

**Comment by ashhitch** [comment]
@Mindgames  I have a test version that should be good to go with Yoast 14 If you fancy testing it:
https://github.com/ashhitch/wp-graphql-yoast-seo/releases/tag/v4.0.0Alpha

**Comment by ashhitch** [comment]
Fixed by #31
